javav2/example_code/cognito/src/main/java/com/example/cognito/CognitoMVP.java

```diff
@@ -208,20 +206,20 @@ public static void verifyTOTP(CognitoIdentityProviderClient identityProviderClie
     //snippet-end:[cognito.java2.token.verify.main]
 
     //snippet-start:[cognito.java2.initiateauth.main]
-    public static InitiateAuthResponse initiateAuth(CognitoIdentityProviderClient identityProviderClient, String clientId, String userName, String password) {
-
+    public static AdminInitiateAuthResponse initiateAuth(CognitoIdentityProviderClient identityProviderClient, String clientId, String userName, String password, String userPoolId) {
         try {
             Map<String,String> authParameters = new HashMap<>();
             authParameters.put("USERNAME", userName);
             authParameters.put("PASSWORD", password);
 
-            InitiateAuthRequest authRequest = InitiateAuthRequest.builder()
+            AdminInitiateAuthRequest authRequest = AdminInitiateAuthRequest.builder()

```

Comment:
I see this is updated to use AdminInitiateAuth, but shouldn't there also be an update to use AdminRespondToAuth? It seems incomplete for the updates in order to match the list: https://github.com/awsdocs/aws-doc-sdk-examples/issues/3305

Reply:
Ok, I see it now.